BlueprintUI/Sources/Layout/Conditionals.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  Conditionals.swift
+//  BlueprintUI
+//
+//  Created by Kyle Van Essen on 9/22/20.
+//
+
+
+extension Element {
+    
+    //
+    // MARK: If / Else
+    //
+    
+    func `if`(
+        _ isTrue : Bool,
+        modify : (inout Self) -> ()
+    ) -> Element
+    {
+        if isTrue {
+            var copy = self
+            modify(&copy)
+            return copy
+        } else {
+            return self
+        }
+    }
+    
+    func `if`(
+        _ isTrue : Bool,
+        onTrue : (inout Self) -> (),
```

Comment:
Actually, should probably convert this to return; or add a second override for return-based.